npm/package.json

```diff
@@ -29,20 +29,18 @@
     "modulePathIgnorePatterns": [
       "<rootDir>/src/.*"
     ],
-    "unmockedModulePathPatterns": [
-      "<rootDir>/node_modules/.*",
-      "<rootDir>/dist/.*"
-    ]
+    "automock": false
```

Comment:
This means we can remove the 'jest.disableAutomock()' call in the test now, right?